SU2_CFD/src/output/CFlowCompOutput.cpp

```diff
@@ -231,7 +231,7 @@ void CFlowCompOutput::SetHistoryOutputFields(CConfig *config){
 
   ///   /// BEGIN_GROUP: HEAT_COEFF, DESCRIPTION: Heat coefficients on all surfaces set with MARKER_MONITORING.
   /// DESCRIPTION: Total heatflux
-  AddHistoryOutput("HEATFLUX", "HF",      ScreenOutputFormat::SCIENTIFIC, "HEAT", "Total heatflux on all surfaces set with MARKER_MONITORING.", HistoryFieldType::COEFFICIENT);
+  AddHistoryOutput("TOTAL_HEATFLUX", "HF",      ScreenOutputFormat::SCIENTIFIC, "HEAT", "Total heatflux on all surfaces set with MARKER_MONITORING.", HistoryFieldType::COEFFICIENT);
```

Comment:
Is this change required for the fix? (thinking about backwards compatibility, documentation, etc.)

Reply:
I think so. According to the AddHistoryOutput function, the first parameter is:

> name - Name for referencing it (in the config file and in the code).

If I understand correctly this parameter is called TOTAL_HEATFLUX everywhere in the code, but not in these 2 files.

Reply:
CHT people @oleburghardt @TobiKattmann, can you give an opinion here, how does this influence existing tutorials, testcases, etc.?

Reply:
Hi, these changes shouldn't affect them as everything is done internally there (the value always comes from `solver->GetTotal_HeatFlux()`, if needed).

So from my side I'd be fine with changing the name. For tutorials and verification it's just about producing and checking the gradient, I've never checked for compatibility with the python framework.
Would be great if it can be integrated into the regression tests some day.. O:-)